### ISSUE TITLE
Don't load PomBase annotations from BioGRID

### ISSUE THREAD

**Issue by pombase-admin** [pombase/pombase-chado#226]
The BioGRID data now includes annotation submitted by PomBase.  Make sure we don't load that.

I've made this into a separate ticket so it doesn't get forgotten.  Original: [chado:#221]

Original comment by: kimrutherford

**Comment by kimrutherford** [comment]
This issue was closed but I don't think it was ever fixed.  The are 2000 interactions that appear to be from BioGRID and have the database_source "POMBASE":

  value  | count 
---------+-------
 BIOGRID | 67487
 PomBase |  1973
 POMBASE |  2101
(3 rows)

query:
```sql
SELECT value, count(feature_relationship_id)
FROM feature_relationshipprop p
WHERE feature_relationship_id IN
    (SELECT feature_relationship_id
     FROM feature_relationship
     WHERE type_id IN
         (SELECT cvterm_id  FROM cvterm WHERE name LIKE 'interacts_%'))
  AND p.type_id IN
    (SELECT cvterm_id FROM cvterm WHERE name = 'source_database')
  AND feature_relationship_id NOT IN
    (SELECT feature_relationship_id
     FROM feature_relationshipprop p2
     WHERE p2.type_id IN
         (SELECT cvterm_id FROM cvterm  WHERE name = 'is_inferred')
       AND p2.value = 'yes')
GROUP BY value; 
```

**Comment by kimrutherford** [comment]
> There are also 2101 interactions in Chado that are from BioGRID but have the source as "POMBASE".

Strangely, until November 2018 there were about 330 POMBASE interactions in the BioGRID tab file.  The number jumped up to 2100 in that release.

That doesn't explain why it the filtering was never implemented though.

**Comment by kimrutherford** [comment]
> That doesn't explain why it the filtering was never implemented though.

I think I've solved the mystery.  I implemented the filtering by ignoring interactions that had the comment "Contributed by PomBase".  I did that because at the time the "Source Database" in the files was "BIOGRID" for interactions supplied by PomBase so looking at the comment so the only way to tell.

In 2015 they removed the comment and changed the "Source Database" to "POMBASE".  I didn't notice the change.

I've now added a filter to ignore to interactions that have a "POMBASE" source.  I'll check tonight's load tomorrow.
